package.json

```diff
@@ -6,9 +6,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bcrypt": "^5.1.0",
```

Comment:
You've still got remnants from adding the backend deps to the front end. These need removing. Did you pull changes to dev, and rebase, before starting this branch.

Reply:
ah i thought i did but I must have done it in the wrong order by the look of it! my bad